src/otx/algo/instance_segmentation/utils/structures/mask/mask_target.py

```diff
@@ -41,7 +43,7 @@ def mask_target(
     """
     cfg_list = [cfg for _ in range(len(pos_proposals_list))]
     mask_targets = map(
-        mask_target_single,
+        mask_target_single,  # type: ignore[arg-type]
```

Comment:
Why is this skip added? Type hints in `mask_target_single` seem there is no change.

Reply:
Updated